src/components/computing-status/use-computing-status.ts

```diff
@@ -68,6 +68,7 @@ function isWorthUpdate(
     if (!updateType) {
         return false;
     }
+    console.log('TEST', invalidations, updateType);
```

Comment:
use the import ComputingType from commons-ui ?

Reply:
done